build.assets/Dockerfile-fips

```diff
@@ -40,6 +62,12 @@ RUN mkdir -p /opt && cd /opt && curl https://go-boringcrypto.storage.googleapis.
     chmod a+w /var/lib && \
     chmod a-w /
 
+# Install libbpf
+RUN mkdir -p /opt && cd /opt && curl -L https://github.com/libbpf/libbpf/archive/refs/tags/v0.3.tar.gz | tar xz && \
```

Comment:
Does this version need to match something somewhere else?  It feels a bit like a "magic number" presently.

Reply:
Let's at least extract it into a makefile variable.